src/pydvl/utils/utility.py

```diff
@@ -110,7 +110,7 @@ def __init__(
         self,
         model: SupervisedModel,
         data: Dataset,
-        scorer: Optional[Union[str, Scorer]] = None,
+        scorer: Optional[Union[str, ScorerCallable]] = None,
```

Comment:
The Scorer implements methods, like `__repr__` that are relied upon in the code, so we can't just change the type. If this is because of your `CSScorer`, you can wrap it in a `Scorer` with the methods provided, without needing to implement those dunders.
```suggestion
        scorer: Optional[Union[str, Scorer]] = None,
```